.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: 🐞 Bug Report
+about: Create a report to help us improve
+title: "[Bug] "
+labels: bug
+assignees: ""
+---
+
+### Summary
+
+<!-- Please provide a summary of the bug -->
+
+#### Expected Behavior
+
+<!--- Tell us what should happen -->
+
+#### Current Behavior
+
+<!--- Tell us what happens instead of the expected behavior -->
+
+### Possible Solution
```

Comment:
I like having summary, expected behavior, and current behavior.

I don't know if we're at a scale where we need all the rest of the sections in that much detail. I'm a little worried about scaring people off. Maybe we just have a single additional section `Anything else we should know?` where we suggest screenshots, steps to reproduce, context, etc.